<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ТЕХНОЛОГИЧЕСКОЕ СПО</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="hrlOscilloscope"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vltWaterfall"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QFrame" name="frameSettings">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>&lt;font color=cyan&gt;Режимы анализатора спектра&lt;/font&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0" colspan="2">
               <widget class="QRadioButton" name="rbRealSpectrum">
                <property name="text">
                 <string>Мгновенный спектр</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lAccumSpectrumDeep">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Глубина:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QRadioButton" name="rbPeakSpectrum">
                <property name="text">
                 <string>Пиковый детектор</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="lAccumSpectrumDeepValue">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>---</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="lPeakSpectrCoeff">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Коэфф.:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="2">
               <widget class="QSlider" name="sliderPeakSpectrCoeff">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>90000</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>100000</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QSlider" name="hslAccumSpectrumDeep">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLabel" name="lPeakSpectrCoeffValue">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>---</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QRadioButton" name="rbSignalEnvelope">
                <property name="text">
                 <string>Огибающая сигнала</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QRadioButton" name="rbAccumSpectrum">
                <property name="text">
                 <string>Накопление</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Частота настройки</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QDoubleSpinBox" name="dsbFreq">
                 <property name="minimum">
                  <double>24.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1766.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>1090.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lFreq">
                 <property name="text">
                  <string>МГц</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbFreqSetup">
                 <property name="text">
                  <string>Установить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
